news/2017-09-24-scorewatch-september-week-2.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Scorewatch: September Week 2"
+date: 2017-09-24 15:00:00 +0200
+---
+
+Scorewatch continues to thrive with more incredible plays from the community! Want to know what -GN, Rafis, Cookiezi, _yu68 and ExGon did to deserve a place in this article? Read on!
```

Comment:
unescaped ``_`` in _yu68 makes the whole article italic

also probably don't want to reference Scorewatch in the opener so directly, maybe mention the socres a bit? like "some great plays on <map 1>" etc, you probably get the idea by now